src/admin.rs

```diff
@@ -41,6 +41,34 @@ impl FluvioAdmin {
             topic_list
         })
     }
+    #[wasm_bindgen(js_name = createTopic)]
+    pub fn create_topic(&mut self, topic_name: String) -> Promise {
```

Comment:
I agree with this, though unfortunately since we haven't spent much time on the admin API we haven't come up with a standard way to do this across different languages. We'll just have to come up with something that works good enough for now